content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md

```diff
@@ -382,8 +382,8 @@ To create the schema manually, do the following:
 
     ```sql
     CREATE USER <RBAC-USERNAME> WITH ENCRYPTED PASSWORD '<RBAC-PASSWORD>';
-    GRANT USAGE ON SCHEMA RPC_RBAC to <RBAC-USERNAME>;
-    GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA RPC_RBAC to <RBAC-USERNAME>;
+    GRANT USAGE ON SCHEMA REST_RBAC to <RBAC-USERNAME>;
+    GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA REST_RBAC to <RBAC-USERNAME>;
```

Comment:
The default schema name here is actually now `RBAC`

Reply:
The 6 `--jdbc-url` in step #4 should have `?currentSchema=RBAC` appended to them in the same way that step #6 has for `CRYPTO`.